src/GraphQL.Tests/QueryTestBase.cs

```diff
@@ -30,7 +30,7 @@ public QueryTestBase()
             Executer = new DocumentExecuter(new TDocumentBuilder(), new DocumentValidator(), new ComplexityAnalyzer());
         }
 
-        public ISimpleContainer Services { get; set; }
+        public TIocContainer Services { get; set; }
```

Comment:
So it allows you to use container-specific APIs like `Scoped` method for `Issue2932_DemoDIGraphType`. On the one hand it is good/correct. On the other hand, should we not write tests working for any DI container? Just as we did with serializers. I understand that it will require refactoring of QueryTestBase.

Reply:
#2937

Reply:
I assume we will track this in #2937.  resolving comment here.